src/server/scripts/Zandalar/AtalDazar/boss_priestess_alun_za.cpp

```diff
@@ -382,8 +382,7 @@ class spell_priestess_alun_za_energy_regen : public AuraScript
     void OnPeriodic(AuraEffect const* aurEff) const
     {
         Unit* target = GetTarget();
```

Comment:
there's no need to make a pointer if you're not checking for null, I guess this is because the holder of the aura is the same as the caster and there's no way there's one without the other in this case?